composer.json

```diff
@@ -22,10 +22,11 @@
     "source": "https://github.com/bryanjhv/slim-session.git"
   },
   "require": {
+    "php": "^7.4 || ^8.0",
     "slim/slim": "^4.0"
   },
   "require-dev": {
-    "phpunit/phpunit": "^7.0"
+    "phpunit/phpunit": "^9.6"
```

Comment:
HOW does it change anything?
Does this SUPPORT slim/slim@**^4.0**.

Reply:
Fundamentally does it change anything? NO, that's why I introduced it as a pedantic PR. 

Functionally it's the same.

What it does do is it gives the library a fresh lick of paint, as otherwise the repo is functionally complete.

I have been using this library as a teaching demonstration of a good framework agnostic lib because of its support for PSR-15. I ultimately began tinkering and thus this PR came to fruition.

Reply:
Well, it *might* change something...
I need to check for PHPUnit 9.6 min PHP version, for example.
So the whole project remains installable with Slim 4.0.0.
The other option is to see Slim 4.x versions usage and accommodate properly (so your change could get merged).

Reply:
I chose `^9.6` because that is what Slim 4.x uses: https://github.com/slimphp/Slim/blob/4.x/composer.json#L68 ultimately this is a dev dependancy however and _can_ be different as it's only installed when working on this library directly and not when required by another.

From its [install documentation](https://docs.phpunit.de/en/9.6/installation.html), PHPUnit 9.6 requires at least PHP 7.3.